infra/gcp/terraform/kubernetes-public/iam.tf

```diff
@@ -27,5 +27,11 @@ module "iam" {
       "serviceAccount:kubernetes-external-secrets@kubernetes-public.iam.gserviceaccount.com",
       "principal://iam.googleapis.com/projects/16065310909/locations/global/workloadIdentityPools/k8s-infra-prow.svc.id.goog/subject/ns/external-secrets/sa/external-secrets",
     ]
+    "roles/dns.admin" = [
```

Comment:
The DNS IAM was created by hand ages ago so I'm managing it by TF. The cert-manager entry was added recently by me to enable #7341